inst-coffee.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="milestone1.css"
+</head>
+<body>
+    <!--Website header - should be the same throughout the pages-->
+    <header class="header">
+        <a class="title-section" href="milestone1.html">
+            <h1 class="title">Arya's Best Cafe</h1>
+        </a>
+        <nav>
+            <a class="navigator" href="milestone1.html">Home</a>
+            <a class="navigator" href="about-me.html">About Me</a>
+        </nav>
+    </header>
+    <!--Content for making coffee specifically-->
+    <main>
+        <div class="recipe-content">
+            <div>
+                <img class="recipe-images" src="https://www.cancer.org/content/dam/cancer-org/images/photographs/single-use/espresso-coffee-cup-with-beans-on-table-restricted.jpg" alt="Picture of coffee"  width="200" height="200">
+            </div>
+            <div class="recipe-content">
+                <h1 class="title">Instant Coffee</h1>
+                <p >
+                    "This instant coffee takes less than 5 minutes to make, and tastes just as refreshing as any other coffee! And, it's the perfect compliment to your banana bread!"
+                <!--Ingredients-->
+                <h1 class="header">Ingredients</h1>
+                <ul>
+                    <li>1.5 cups of milk, preferably 2% or whole milk</li>
+                    <li>2 teaspoons granulated sugar
+                    <li>1.5 teaspoons coffee powder</li>
+                </ul>
+            </div>
+        </div>
+        <!--Instructions-->
+        <h1 class="header">Instructions</h1>
+        <ol class="recipe-content">
+            <li>Heat up the milk for about 45 seconds in a microwave.</li>
+            <li>Add the coffee mix and the sugar and stir completely.</li>
+            <li>Microwave the milk, powder, and sugar for another 30 seconds and stir when finished.</li>
+            <li>Serve and enjoy!</li>
+        </ol>
```

Comment:
Not sure why you have your instructions outside of your recipe-content div while the ingredients are inside...